testserver/bokchoy_settings.py

```diff
@@ -168,3 +171,6 @@
 NOTIFICATION_CHANNEL_PROVIDER_TYPE_MAPS = {
     '*': 'durable',  # default global mapping
 }
+
+
+ALLOWED_HOSTS = ['*']
```

Comment:
Why do we need to add allowed hosts?

Reply:
I'll remove it. Not sure how it was added.